.github/workflows/mac.yml

```diff
@@ -206,13 +199,11 @@ jobs:
           path: target.tar.gz
 
   wpt-2020:
-    if: ${{ inputs.wpt-layout == '2020' || inputs.wpt-layout == 'all' }}
```

Comment:
We shouldn't remove this if condition. They will be updated as part of a different issue.